src/datasets/bigquery/clients_last_seen/reference.md

```diff
@@ -60,6 +60,41 @@ The data is available in Re:dash and BigQuery. Take a look at this full running
 
 # Data Reference
 
+## Field Descriptions 
+The [“*_bits” fields](../../../cookbooks/clients_last_seen_bits.md) store the relevant activity of a client in a 28 day 
+window, as a 28 bit integer. 
+For each bit, a 1 corresponds to the specific activity occurring on that day.
+
+### Activity Segment/User State/Core Active Specific
```

Comment:
do we want to add a link to relevant chapter on user segments here?

Reply:
Yep. added.